tracing/opentelemetry/src/test/java/io/helidon/tracing/opentelemetry/BaggageTest.java

```diff
@@ -53,4 +57,34 @@ void testBadBaggage() {
         assertThrows(NullPointerException.class, () -> span.baggage("key", null));
         assertThrows(NullPointerException.class, () -> span.baggage(null, null));
     }
+
+    /**
+     * Test for: https://github.com/helidon-io/helidon/issues/6970
+     */
+    @Test
+    @Disabled // temporary disabled
+    void baggageCanaryMinimal() {
+        final var tracer = io.helidon.tracing.Tracer.global();
+        final var span = tracer.spanBuilder("baggageCanaryMinimal").start();
+        try {
+            // Set baggage and confirm that it's known in the span
+            span.baggage("fubar", "1");
+            assertThat("1", is(span.baggage("fubar").orElseThrow()));
+
+            // Inject the span (context) into the consumer
+            final var consumer = HeaderConsumer
+                    .create(new TreeMap<>(String.CASE_INSENSITIVE_ORDER));
+            tracer.inject(span.context(), HeaderProvider.empty(), consumer);
+
+            // Confirm that baggage was NOT propagated (the bug)
```

Comment:
This comment was appropriate for the standalone test case showing the issue, but as a test for the fix remove 'NOT'

Reply:
This is outdated.